site/public/js/office-hours-queue.js

```diff
@@ -0,0 +1,76 @@
+//this helps update the frontend when the page refreshes because without this the sort icon would reset and the sort state would not
+/* eslint prefer-arrow-callback: [ "error", { "allowNamedFunctions": true } ] */
+document.addEventListener('DOMContentLoaded', () => {
+    const sortIndicator = document.getElementById('sort-indicator');
+    let sortState = localStorage.getItem('sort-indicator');
+
+    if (sortState === null) {
+        sortState = 'off';
+        localStorage.setItem('sort-indicator', sortState);
+    }
+
+    if (sortState === 'up') {
+        sortIndicator.innerHTML = '<i class="fas fa-sort-up"></i>';
```

Comment:
Using `.innerHTML` opens the possibility for XSS attacks to pop up in the future.  You could just use jQuery to change the class instead rather than adding some HTML.  Apply throughout.